### DISCUSSION TITLE
Snowden on AI Explosion and Data Privacy

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#26] [Extra Credit]
Watch `Edward Snowden and Ben Goertzel on the AI Explosion and Data Privacy`: 

- https://www.youtube.com/watch?v=4H_iWpWG_c0

Respond to ANY or all of the prompts below in 2-3 sentences:

### How do you think we can apply the human aspects of personal growth, ethics, and development to Artificial Intelligence?

**Comment by danigold1020** [comment]
As discussed in the video, both humans and computers make observations, absorb information from these observations, and learn by doing so. We need to ensure that an extra level of ethics is applied to the decision-making process involved in machine learning algorithms, which in a sense, can make  machines perform better than humans as they can act more ethically (a quality that sometimes lacks in human actions) and personally develop over time. This can make machines perform better than humans, as suggested in the video, as they act more ethically with a deeper understanding of the consequences of their actions and human emotion (e.g. pain, surprise, etc.).
